code/modules/mob/living/simple_animal/friendly/robot_customer.dm

```diff
@@ -0,0 +1,80 @@
+///Robot customers
+/mob/living/simple_animal/robot_customer
+	name = "space-tourist bot"
+	maxHealth = 50000000 //go fuck yourself
+	health = 50000000
+	desc = "I wonder what they'll order..."
+	icon = 'icons/mob/tourists.dmi'
+	icon_state = "amerifat"
+	icon_living = "amerifat"
+	///Override so it uses datum ai
+	can_have_ai = FALSE
+	AIStatus = AI_OFF
+	del_on_death = TRUE
+	mob_biotypes = MOB_ROBOTIC|MOB_HUMANOID
+	ai_controller = /datum/ai_controller/robot_customer
+	unsuitable_atmos_damage = 0
+	minbodytemp = 0
+	maxbodytemp = 1000
+	var/clothes_set = "amerifat_clothes"
+	var/datum/atom_hud/hud_to_show_on_hover
```

Comment:
These alt appearances need to be deleted. From what i can see they'll be kept around even after the customer deletion.